source/conf.py

```diff
@@ -122,3 +129,25 @@
 # -- Internationalization options ---------------------------------------------
 locale_dirs = ['locales/']
 gettext_compact = False
+
+# -- Substitutions ------------------------------------------------------------
+
+# Get the YYYY year from the current commit in the user's guide.
+try:
+    commit_year = subprocess.run(
+        ['git', 'show', '-s', '--format=%cs', 'HEAD'],
+        capture_output=True).stdout.decode('ASCII').strip().split('-')[0]
+except subprocess.CalledProcessError:
+    # If there's an error, default to the current year.
+    commit_year = datetime.datetime.now().year
+
+# Here we expose variables for use in our RST pages.
+rst_prolog = f"""
```

Comment:
I'm not certain what `rst_prolog` is for, but I was wondering if `|version| replace:: {version}` should also be in this list, since we're exposing it to use in the RST? I could be interpreting this wrong.

Reply:
It looks like `rst_prolog` is inserted at the beginning of every source file as it is being read, so it allows us to do these kinds of global substitutions. https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-rst_prolog

It does look like this is a very general bit of RST, though, so we could probably insert any text or RST we wanted to!